Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -102,36 +102,30 @@ final class DemoMerchantAPI {
         _ = try await data(for: urlRequest)
     }
 
-    /// This function fetches an access token to initialize any module of the SDK
+    /// This function fetches a clientID to initialize any module of the SDK
     /// - Parameters:
     ///   - environment: the current environment
-    /// - Returns: a String representing an access token
-    /// - Throws: an error explaining why process order failed
-    public func getAccessToken(environment: Demo.Environment) async -> String? {
-        if let injectedAccessToken = InjectedValues.accessToken {
-            return injectedAccessToken
-        }
-        guard let token = self.accessToken else {
-            self.accessToken = await fetchAccessToken(environment: environment)
-            return self.accessToken
-        }
-        return token
+    /// - Returns: a String representing an clientID
+    /// - Throws: an error explaining why fetch clientID failed
+    public func getClientID(environment: Demo.Environment) async -> String? {
+       
+        guard let clientID = self.clientID else {
+            self.clientID = await fetchClientID(environment: environment)
```

Comment:
Do we want to persist clientID in between the checkout sessions or fetch it every time from merchant server?